openedx/core/djangoapps/agreements/readme.rst

```diff
@@ -0,0 +1,23 @@
+############
+Agreements App
+############
+
+Status: Active
+
+Purpose
+=======
+The intention of this app is to host logic and data associated with the agreements learners needs to "sign" or "confirm".
+These agreements includes, but not limited to, Honor Code to affirm they are who they say they are.
+
+
+Responsibilities
+================
+
+The app intend to store all logic associated with agreements edX needs to display and record for either learners or educators.
+Currently, the app only store the Honor Code signature for learners. 
+In order to reinforce academic integrity, learners will be required to agree to the honor code before entering the first assignment on paid track.
+
+Glossary
+========
+* *Signature*: It's a record of any time to store user's action against an agreement or code.
```

Comment:
NIT: I think the wording here is a little unclear, maybe "A record denoting the time of a user's action against an agreement or code" ?

Reply:
This is a mistake I made. Thank you for your suggestion. Updated the glossary section.